cli/install/install.go

```diff
@@ -1204,6 +1206,8 @@ func installTarantool(binDir string, incDir string, installCtx InstallCtx,
 	}
 
 	log.Infof("Done.")
+	log.Infof("Tarantool executable is installed to: %s/%s", binDir, versionStr)
```

Comment:
It is possible to forget to update this log message if the logic of the copyBuilded* function is changed, as we generate these paths twice (in the function and in the log message). It would be more appropriate to provide a message regarding the installed executable, even if the symbolic link creation fails.